services/app/app/components/views/quest/cardtemplates/decision/Scenarios.tsx

```diff
@@ -23,6 +68,26 @@ const scenarios: ScenarioType[] = [
     nonevent: {type: 'INTERRUPTED', text: 'In the chaos of combat, your fancy footwork fails to fool the fiend. No loot for you, this time.', instructions: []},
     retry: {type: 'RETRY', text: 'The enemy dodges... but you see another chance!', instructions: []},
   },
+  simpleScenario(
+    'Dark',
+    'Athletics',
+    'An enemy has let their guard down... now is your chance to strike!',
+    'Your attack hits true!',
+    ['Deal 2 damage to one target.', 'Reduce damage taken this round by 1'],
+    'Your attack goes wide; you leave yourself open to retaliation!',
+    ['The last adventurer to roll for this check takes 1 damage.'],
+    'The enemy closes their guard before you get a good hit in.',
+    'Your attack goes wide; but there\'s another chance!'),
+  simpleScenario(
+    'Dark',
+    'Athletics',
+    'You see some loose dust on the ground - you could use it to blind the enemy temporarily...',
+    'You scoop up a handful of the debris and toss it at the nearest enemy, which starts clawing at their eyes in rage!',
+    ['Enemies deal 2 less damage next round'],
+    'As you crouch down, an enemy sees you drop your guard and strikes!',
+    ['All adventurers that rolled for the check take +1 damage from any damage they take this round'],
```

Comment:
`+1 damage from enemies next round`

^^ which, also, to clarify... are we calling the forthcoming round this round or next round?

Reply:
done